docs/Api/DefaultApi.md

```diff
@@ -954,6 +1017,63 @@ Name | Type | Description  | Notes
 [[Back to Model list]](../../README.md#models)
 [[Back to README]](../../README.md)
 
+## `findMotionCapture()`
+
+```php
+findMotionCapture($motion_capture_id): \Onfido\Model\MotionCapture
+```
+
+Retrieve motion capture
+
+### Example
+
+```php
+<?php
+require_once(__DIR__ . '/vendor/autoload.php');
+
+$config = Onfido\Configuration::getDefaultConfiguration();
+$config->setApiKey('Authorization', 'token=' . 'YOUR API KEY');
+$config->setApiKeyPrefix('Authorization', 'Token');
+
+$apiInstance = new Onfido\Api\DefaultApi(
+    // If you want use custom http client, pass your client which implements `GuzzleHttp\ClientInterface`.
+    // This is optional, `GuzzleHttp\Client` will be used as default.
+    new GuzzleHttp\Client(),
+    $config
+);
+$motion_capture_id = 'motion_capture_id_example'; // string | The motion capgture’s unique identifier.
+
+try {
+    $result = $apiInstance->findMotionCapture($motion_capture_id);
+    print_r($result);
+} catch (Exception $e) {
+    echo 'Exception when calling DefaultApi->findMotionCapture: ', $e->getMessage(), PHP_EOL;
+}
+```
+
+### Parameters
+
+Name | Type | Description  | Notes
+------------- | ------------- | ------------- | -------------
+ **motion_capture_id** | **string**| The motion capgture’s unique identifier. |
```

Comment:
typo on the word capture

Reply:
Fixed in all occurrences :pray: Thank you.